### DISCUSSION TITLE
GrapeJS Templates

### DISCUSSION THREAD

**Discussion by lifeofsa** [GrapesJS/grapesjs#5121] [Q&A] [answered]
I want to add the pre-defined templates in the grapesjs but I couldn't can anyone help me please?

**Comment by artf** [comment, marked as answer]
This [plugin](https://github.com/Ju99ernaut/grapesjs-template-manager) is a good example how to implement a template manager

**Reply by lifeofsa** [reply]
I have stored my templates on mysql DB and I have to retrieve it from their
